src/components/tutorial/Option.astro

```diff
@@ -40,6 +40,10 @@ const { isCorrect } = Astro.props as Props;
 		appearance: none;
 	}
 
+	input[type='radio']:hover {
```

Comment:
Thanks @ralacerda! No need to target `:hover` on this style — we can just tag this on to the `input[type='radio']` styles:

```suggestion
```